packages/google_maps_flutter/google_maps_flutter/example/ios/RunnerTests/MockGMSMapView.h

```diff
@@ -0,0 +1,11 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@import GoogleMaps;
+
+@interface MockGMSMapView : GMSMapView
```

Comment:
You should probably call this `PartiallyMockedMapView` or something along those lines; the assumption for a mock object is not that it is the real object but with some overridden behavior, but that it has no behavior other than what it is given in test setup.